packages/cf-component-box/src/Box.js

```diff
@@ -0,0 +1,117 @@
+import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
+
+const styles = (props) => ({
+  margin: props.margin || props.theme['margin'],
+  marginTop: props.marginTop || props.theme['marginTop'],
+  marginBottom: props.marginBottom || props.theme['marginBottom'],
+  marginLeft: props.marginLeft || props.theme['marginLeft'],
+  marginRight: props.marginRight || props.theme['marginRight'],
+
+  padding: props.padding || props.theme['padding'],
+  paddingTop: props.paddingTop || props.theme['paddingTop'],
+  paddingBottom: props.paddingBottom || props.theme['paddingBottom'],
+  paddingLeft: props.paddingLeft || props.theme['paddingLeft'],
+  paddingRight: props.paddingRight || props.theme['paddingRight'],
+
+  backgroundColor: props.backgroundColor || props.theme['backgroundColor'],
+  display: props.display || props.theme['display'],
+  position: props.position || props.theme['position'],
+  top: props.top || props.theme['top'],
+  bottom: props.bottom || props.theme['bottom'],
+  left: props.left || props.theme['left'],
+  right: props.right || props.theme['right'],
+
+  borderWidth: props.borderWidth || props.theme['borderWidth'],
+  borderColor: props.borderColor || props.theme['borderColor'],
+  borderStyle: props.borderStyle || props.theme['borderStyle'],
+
+  borderTopWidth: props.borderTopWidth || props.theme['borderTopWidth'],
```

Comment:
I wrote some placeholder code, but this didn't quite pan out because of our numbers :/

Our current baseline is 22.5 pixels, which really sucks. It's also encoded as a sass variable. I think there's some benefit to encoding this in a single source of truth, perhaps in yet another package that exports a simple data structure.